README.md

```diff
@@ -15,18 +15,20 @@ yarn add --dev @swc/core @swc/register
 ```
 
 ### Usage
-```bash
-require("@swc/register");
-```
-
-All subsequent files required by node will be transformed by swc. You can also
-call @swc/register directly from command line.
 
-```bash
-swc-node <filename>
-node -r '@swc/register' <filename>
```

Comment:
As the automatic hook is being removed in #14, `node -r '@swc/register'` is no longer supported.

Babel has a dedicated package `@babel/node` (`node -r @swc/node`). swc should use a dedicated package, too.

Reply:
> As the automatic hook is being removed in #14, node -r '@swc/register' is no longer supported.
> Babel has a dedicated package @babel/node (node -r @swc/node). swc should use a dedicated package, too.

This does mean we need a custom CLI?
While reading again before merging into the main repository, I found that this package would be useless.